doc/source/dev/roadmap-detailed.rst

```diff
@@ -71,49 +59,67 @@ add new benchmarks, however running the benchmarks is not very intuitive.
-Windows build issues
-````````````````````
-SciPy critically relies on Fortran code. This is still problematic on Windows.
-There are currently only two options: using Intel Fortran, or using MSVC +
-gfortran.  The former is expensive, while the latter works (it's what we use
-for releases) but is quite hard to do correctly.  For allowing contributors and
-end users to reliably build SciPy on Windows, using the Flang compiler looks
-like the best way forward long-term.
+Use of Pythran
+``````````````
+Pythran is still an optional build dependency, and can be disabled with
+``SCIPY_USE_PYTHRAN=0``. The aim is to make it a hard dependency - for that to
+happen it must be clear that the maintenance burden is low enough (Meson will
+help here, because it removes the monkey patching that is now done to enable
+Pythran).
 
 
 Continuous integration
 ``````````````````````
-Continuous integration is in good shape, it currently covers the Windows, macOS
-and Linux, ARM64 and ppc64le platforms, as well as a range of versions of our
-dependencies and building release quality wheels.
+Continuous integration currently covers 32/64-bit Windows, macOS on x86-64, and
+32/64-bit Linux on x86, and Linux on aarch64 - as well as a range of versions
+of our dependencies and building release quality wheels. Reliability of CI has
+not been good recently (H2 2021), due to the large amount of configurations to
+support and some CI jobs needing an overhaul. We aim to reduce build times,
+improve caching, move more jobs to GitHub Actions, drop TravisCI and Appveyor
+in the `scipy-wheels repo <https://github.com/MacPython/scipy-wheels>`_,
+move from ``multibuild`` to ``cibuildwheel`` for building wheels for releases,
+and make the set of configurations in CI jobs more orthogonal.
 
 
 Size of binaries
 ````````````````
-SciPy binaries are quite large (e.g. an unzipped manylinux wheel for 1.4.1 is
-91 MB), and this can be problematic - for example for use in AWS Lambda, which
-has a 250 MB size limit. We aim to keep binary size as low as possible; when
-adding new compiled extensions, this needs checking. Stripping of debug symbols
-in ``multibuild`` can likely be improved (see `this issue
-<https://github.com/matthew-brett/multibuild/issues/162>`__).
+SciPy binaries are quite large (e.g. an unzipped manylinux wheel for 1.7.3 is
+39 MB on PyPI and 122 MB after installation), and this can be problematic - for
+example for use in AWS Lambda, which has a 250 MB size limit. We aim to keep
+binary size as low as possible; when adding new compiled extensions, this needs
+checking. Stripping of debug symbols in ``multibuild`` can perhaps be improved
+(see `this issue <https://github.com/matthew-brett/multibuild/issues/162>`__).
+An effort should be made to slim down where possible, and not add new large
+files. In the future, things that are being considered (very tentatively) and
+may help are separating out the bundled` ``libopenblas`` and removing support
+for ``long double``.
```

Comment:
Are we still targeting `multibuild` stripping changes side-by-side with `cibuildwheel` migration?

Unbundling openblas from wheels => proposing runtime linalg backend selection infrastructure?

Reply:
Good question, probably not. Maybe let's keep it on the radar though until we actually find the energy to move to `cibuildwheel`.